client/my-sites/people/viewers-list/invite-button.jsx

```diff
@@ -0,0 +1,13 @@
+import { Button, Gridicon } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+
+const InviteButton = ( { siteSlug, translate } ) => {
```

Comment:
The content of this component appears to exist in 3 components already:

https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/client/my-sites/people/people-list-section-header/index.jsx#L70

https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/client/my-sites/people/followers-list/followers.jsx#L97

https://github.com/Automattic/wp-calypso/blob/9fb7ac149e55361c0c3a9639ad0caa5e15fc88b0/client/my-sites/people/people-invites/index.jsx#L220

and we seem to be creating a separate one without reusing it in those places.

Care to reuse that component where possible?

Reply:
Sounds good to put it there 👍

Reply:
Consolidated the buttons in [81e59cb](https://github.com/Automattic/wp-calypso/pull/63748/commits/81e59cb3e2b1a72e4ee5f5c353f5283d22bfe7ce).